paper/src/main/resources/config.yml

```diff
@@ -202,7 +202,7 @@ hacks:
   PortalModifyHack:
     enabled: false
     targetWorld: world_nether
-    homeWorld: world
+    homeWorld: worldUUID
```

Comment:
How come we changed this line in this PR? world matching based on name is a bad idea yes however this would become a broken by default hack if used anywhere else

Reply:
I ... haven't... oh my intellij auto refactoring changed this because I had a variable called "world."